### PR TITLE
remove deprecated dependency assert_cli

### PR DESCRIPTION
**Pull request by andjo403**

### CODE REVIEW

File: src/test/mod.rs [hunk 1]
```diff
@@ -973,11 +971,10 @@ fn rustfmt() -> PathBuf {
...
-        temp_file.path.to_str().unwrap(),
-    ])
-    .succeeds()
-    .unwrap();
+
+    Command::new(rustfmt().to_str().unwrap())
+        .arg("--check")
+        .arg(temp_file.path.to_str().unwrap())
+        .status()
+        .expect("check failed");
```
Comment (scampi) [inline src/test/mod.rs]: nitpick: `check failed` feels generic, maybe a message like `run with check option failed` would be better.

**Comment by andjo403** [comment]
thanks for the fast feedback

**Comment by andjo403** [comment]
the faults in ci is:
limited_independence_of_irrelevant_alternatives runs for a long time in cargo integration see rust-lang/cargo#6258 for tracing of this fault

and futures-test-preview do not compile

**Comment by Eh2406** [comment]
@nrc merged without bors?

**Comment by Eh2406** [comment]
Sorry thought I was still on /cargo

**Pull request merged**